app/quizzes/[id]/(trying)/not-found.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import TypeTime from '@/assets/images/type-time.png';
+
+export default function NotFound() {
+  return (
+    <>
+      <div className="mt-40 flex max-w-md flex-col items-center gap-4">
+        <Image src={TypeTime} alt="로고" priority width={160} height={160} />
+        <p className="text-lg">페이지를 찾을 수 없습니다.</p>
+        <Link
+          href="/"
+          className="rounded bg-blue-primary px-4 py-2 text-2xl font-bold text-white"
```

Comment:
mt-8, text-lg 정도로 하면 좋을 것 같아요!

```suggestion
          className="mt-8 rounded bg-blue-primary px-4 py-2 text-lg font-bold text-white"
```

![image](https://github.com/grow-playground/type-time/assets/97094709/ac03f633-1c5e-4b32-8e5b-8d9f6df9386a)